frontend/app/src/components/table-filter/TableFilter.vue

```diff
@@ -65,12 +65,37 @@ const selectedMatcher = computed(() => {
 
 const usedKeys = computed(() => get(selection).map(entry => entry.key));
 
+const suggestionBeingEdited = ref<Suggestion | null>(null);
```

Comment:
```suggestion
const suggestionBeingEdited = ref<Suggestion>();
```

any reason to default to null?